src/main/groovy/org/transmartproject/batch/clinical/db/objects/Tables.groovy

```diff
@@ -18,6 +18,8 @@ final class Tables {
     public static final String MODIFIER_DIM      = 'i2b2demodata.modifier_dimension'
     public static final String MODIFIER_METADATA = 'i2b2demodata.modifier_metadata'
     public static final String MODIFIER_DIM_VIEW = 'i2b2demodata.modifier_dimension_view'
+    public static final String BIO_EXPERIMENT    = 'biomart.bio_experiment'
+    public static final String SECURE_OBJECT     = 'searchapp.search_secure_object'
```

Comment:
@cataphract Code looks fine for me. I'm wondering how above tables are related with security or with "private study loaded with tm-batch, cannot be managed in admin tab" bug. Could you please write commit description that shortly will explain the reasoning behind such fix, how these tables are related with security.